### DISCUSSION TITLE
Adding NPI type 2 to provider object

### DISCUSSION THREAD

**Discussion by vvoitkun** [CMSgov/price-transparency-guide#495] [Q&A] [answered]
I am trying to understand how we would add NPI type 2. Example I want to ask is, let say we have a practice where we have both physicians and Nurse practitioner both take office visits. Physician get paid $100 for a visit and nurse practitioner get $75. Both have the same NPI type 2 as they are part of the same group. Currently it will create 2 blocks in MRF file.
Would we add that NPI type 2 to both blocks so that using that NPI someone would come back with 2 prices?

**Comment by shaselton-usds** [comment, marked as answer]
Do you have the NPI type 1s that are under the NPI type 2? This is how you could break up the reporting.

**Reply by vvoitkun** [reply]
I understand that having NPI type 1 would still break it to 2 blocks, but I am trying to understand, you would want us to add that NPI type 2  to all blocks we have for that practice?

**Reply by shaselton-usds** [reply]
Ah, I see -- I wasn't sure if you had the Type 1s (sounds like you do). Yes, both must be added. You can review that [commit here.](https://github.com/CMSgov/price-transparency-guide/commit/faa673043c21e181ff61d38a3dfd4f044b82b14e)
